mne/epochs.py

```diff
@@ -1600,6 +1599,24 @@ def _drop_log_stats(drop_log, ignore=('IGNORED',)):
     return perc
 
 
+def _dep_eeg_ref(add_eeg_ref, current_default=True):
+    """Helper for deprecation"""
+    if current_default is True:
+        if add_eeg_ref is None:
+            add_eeg_ref = True
+            warn('add_eeg_ref will default to True in 0.13 and False in 0.14, '
```

Comment:
So you'd prefer `add_eeg_ref defaults to True in 0.13 and will default to False in 0.14`?